include/chimera/configuration.h

```diff
@@ -216,6 +216,12 @@ class CompiledConfiguration
     bool Render(const std::shared_ptr<chimera::mstch::Function> context);
     bool Render(const std::shared_ptr<chimera::mstch::Variable> context);
 
+    /**
+     * Renders the top-level mstch template where the filename is specified by
+     * Configuration::SetOutputModuleName().
+     */
+    void Render();
```

Comment:
Hmm, I see that there are other functions called `Render()`, we should probably pick a different name here.
```suggestion
    void Serialize();
```

Reply:
I considered that but didn't because the other Render() functions (including the protected one) indeed have similar purposes. The only difference is that this `Render()` generates the top-level bindings while others generate sub bindings for variables, functions, class, etc.

I would be fine to rename this to `Serialize()` as well.

Reply:
Ok, we can leave it alone then, this seems fine.